server/provision_utility.py

```diff
@@ -225,4 +166,3 @@ def provisionDatabase():
     _provisionImages()
     _provisionSegmentationGroups()
     _provisionStudies()
-    _provisionDefaultFeatureset()
```

Comment:
Why remove the default "Basic" featureset? I can see that it helps for testing, but is there another reason to change this?